app/controllers/orders_controller.rb

```diff
@@ -22,8 +23,7 @@ def create
   end
 
   def update
-    @order = @order.merge(dispatcher_id:
-                          current_dispatcher.id) if current_dispatcher.present?
+    @order.dispatcher_id = current_dispatcher.id if current_dispatcher.present?
     respond_to do |format|
       if @order.update(order_params)
```

Comment:
I delete this expression, cause it doesn't work. It returns an error - NoMethodError